### PR TITLE
Add "maxFilesize" param to backend form widget "fileupload"

### PR DESCRIPTION
**Pull request by webmaxx**
Add option "maxFilesize" for backend widget "fileupload".
256MB is used by default, and there is no way to override it.

Use:
```
fields:
        video:
            tab: Content
            label: "Video File"
            type: fileupload
            mode: file
            fileTypes: mp4
            maxFilesize: 1024
```

### CODE REVIEW

**Comment by LukeTowers** [comment]
@webmaxx will need a PR to the docs repo documenting it too. Also, can we have the default automatically load from http://php.net/manual/en/ini.core.php#ini.upload-max-filesize?

**Comment by webmaxx** [comment]
> This pull request has been made to the wrong branch. Please review [the contributing guidelines](https://github.com/octobercms/october/blob/master/.github/CONTRIBUTING.md#pull-requests) as all PRs need to be made to the develop branch.
> 
> I'll fix it for you this time, but please ensure you make any future PRs to the develop branch, not the master branch.

OK. I will consider for the future.

**Comment by webmaxx** [comment]
> will need a PR to the docs repo documenting it too.
OK.

> Also, can we have the default automatically load from http://php.net/manual/en/ini.core.php#ini.upload-max-filesize?
Good idea.

**Comment by LukeTowers** [comment]
@webmaxx ping me when you're done with the ini loading the default max filesize. Note that you'll need to update your docs PR to reference the fact that it defaults to that value

**Comment by webmaxx** [comment]
> @webmaxx ping me when you're done with the ini loading the default max filesize. Note that you'll need to update your docs PR to reference the fact that it defaults to that value

Success

**Comment by LukeTowers** [comment]
@w20k could you review this when able?

**Comment by w20k** [comment]
@LukeTowers  on it!

**Comment by w20k** [comment]
@LukeTowers - LGTM, testing is also done. Took me a while, sorry about that!
If you have something to add, feel free ;)

Screenshots:
Default value = `32MB`
Screenshot 2019-07-04 at 16 34 21
Screenshot 2019-07-04 at 16 39 20

Custom bigger than default: `120MB` (doesn't work, which I guess is logical)
Screenshot 2019-07-04 at 16 50 28
Screenshot 2019-07-04 at 16 45 51
Screenshot 2019-07-04 at 16 46 56

Custom lower than default: `10MB`
Screenshot 2019-07-04 at 16 54 24

**Comment by LukeTowers** [comment]
@webmaxx @w20k can we add some logic to this so that if the FileUpload formwidget sets a max filesize that is over `upload_max_filesize` an ApplicationException is thrown telling the developer exactly what is wrong and how to fix it?

**Comment by w20k** [comment]
@LukeTowers - sure, should be easy to implement. Will be back on Sunday and look at it if @webmaxx won't available!

**Comment by webmaxx** [comment]
> @LukeTowers @w20k Ok. Added

File: modules/backend/formwidgets/FileUpload.php [hunk 1]
```diff
@@ -1,6 +1,7 @@
 <?php namespace Backend\FormWidgets;
 
 use Db;
+use Lang;
```
Comment (LukeTowers) [inline modules/backend/formwidgets/FileUpload.php]: Why was this added?

Comment (webmaxx) [reply modules/backend/formwidgets/FileUpload.php]: I wanted to make a message with the translation, but changed my mind. Сorrected.
